### ISSUE TITLE
Issues when attempting install for "The 1.12.2 Pack"

### ISSUE THREAD

**Issue by KernelDeimos** [cdbbnnyCode/modpack-installer#8]
Hello,

I'm trying to install "The 1.12.2 Pack" (the Curseforge version) on Ubuntu 20.04.

link: https://www.curseforge.com/minecraft/modpacks/the-1122-pack/files/3359972

I'm using `--manual` because `ForgeHack.java` fails (`Predicate` class not found)

After installing, it looks successful but says `Forge did not install correctly`. `minecraft-launcher` actually shows a `1.12.2` modpack has been installed, but it only contains Forge and doesn't contain any of the mods.

I highly suspect this has something to do with the format of the zip file, but I haven't been able to figure this out myself yet

**Comment by ctron** [comment]
I am having the same issue with: https://www.curseforge.com/minecraft/modpacks/better-minecraft-modpack/files/3596605

**Comment by ctron** [comment]
I got one step closer by selecting the "correct" directory in the dialog that opens with the `--manual` option. It needs to be `pack/<yourpack>/.minecraft`.

It downloads the addons, but then fails with another error:

```
Traceback (most recent call last):
  File "/home/jreimann/apps/modpack-installer/install.py", line 196, in <module>
    main(args.zipfile, args.forge_disable)
  File "/home/jreimann/apps/modpack-installer/install.py", line 140, in main
    mods = mod_download.main(packdata_dir + '/manifest.json', '.modcache')
  File "/home/jreimann/apps/modpack-installer/mod_download.py", line 70, in main
    loop.run_until_complete(future)
  File "/usr/lib64/python3.10/asyncio/base_events.py", line 641, in run_until_complete
    return future.result()
  File "/home/jreimann/apps/modpack-installer/mod_download.py", line 56, in download_mods_async
    for resp in await asyncio.gather(*tasks):
  File "/usr/lib64/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/jreimann/apps/modpack-installer/mod_download.py", line 32, in fetch_mod
    file_type = project_info['websiteUrl'].split('/')[4] # mc-mods or texture-packs
TypeError: 'NoneType' object is not subscriptable
```

**Comment by cdbbnnyCode** [comment]
@ctron that looks like a separate problem; could you make a new issue so I can keep track of it separately?
@KernelDeimos did you make sure to point the forge installer to the right directory?

**Comment by cdbbnnyCode** [comment]
@KernelDeimos your issue has been fixed in v2.2-beta:
1) Forge is now installed to the user `.minecraft` directory, reducing issues with manual installation
2) Manual installation should not be necessary as the auto-installer has been updated to work with the latest Forge installer.
